enhancements/ingress/admin-configured-ca-bundle.md

```diff
@@ -0,0 +1,304 @@
+---
+title: admin-configured-ca-bundle
+authors:
+  - "@bhb"
+reviewers:
+  - "@candita" ## reviewer for cluster-ingress-operator component
+  - "@Miciah" ## reviewer for cluster-ingress-operator component
+  - "@tgeer" ## reviewer for CFE
+approvers:
+  - "@candita" ## approval from cluster-ingress-operator component
+  - "@Miciah" ## approval from cluster-ingress-operator component
+  - "@jerpeter1" ## approval from CFE
+api-approvers:
+  - None
+creation-date: 2023-11-02
+last-updated: 2023-12-06
+tracking-link:
+  - https://issues.redhat.com/browse/RFE-2182
+  - https://issues.redhat.com/browse/OCPSTRAT-431
+  - https://issues.redhat.com/browse/NE-761
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Support for admin configured CA trust bundle in Ingress Operator
+
+## Summary
+
+This enhancement allows an OpenShift cluster administrator to configure OpenShift router with custom
+trusted Certificate Authorities (CAs) to use for verifying services' certificates of re-encrypt routes.
+The enhancement proposes configuring custom trusted Certificate Authorities (CAs) which also includes the
+OpenShift service CA certificate for an IngressController to be used for certificate validation on
+re-encrypt routes.
+
+## Motivation
+
+When an OpenShift administrator creates a route with the re-encrypt termination type, they have to also
+provide a CA certificate for that route if it uses a custom CA instead of OpenShift's service CA for
+certificate validation. In addition, when the same CA is used by the administrator for signing several
+services' certificates it is time-consuming and error-prone to configure the same CA certificate in every
+created route object. Instead, this enhancement proposes a simpler alternative: in which an administrator
```

Comment:
this one really is administrator and specifically "cluster-administrator".

Notice that by  using the incorrect term above, this reads as though it's the same person. But when using the correct terms, it's clear that we're talking about different people.